Gemfile

```diff
@@ -72,6 +72,7 @@ gem 'aws-sdk-core', '~> 2.2.15'
 # Optional Services.
 gem 'omniauth-37signals'          # BasecampAgent
 gem 'omniauth-wunderlist', github: 'wunderlist/omniauth-wunderlist', ref: 'd0910d0396107b9302aa1bc50e74bb140990ccb8'
+gem 'omniauth-lifx'
```

Comment:
I think it's fine to have it enabled by default, the gem itself it really small and does not add any other dependencies. Adding `require: false` and explicitly requiring it in the controller where it is needed would help the memory usage of the background processors.